server/service/individualServices/ReadInventoryData.js

```diff
@@ -265,18 +310,504 @@ async function formulateEquipmentInfo(equipmentInfoList) {
+* @param {Object} ltpStructure ControlConstruct provided from cache
+* @param {String} uuidUnderTest Identifier of the air-interface under test
+* @param {Object} requestHeaders Holds information of the requestHeaders like Xcorrelator , CustomerJourney,User etc.
+* @param {Integer} traceIndicatorIncrementer traceIndicatorIncrementer to increment the trace indicator
+* @returns {Object} return values of installed-firmware list and traceIndicatorIncrementer
+*/
+async function FetchConfiguredGroupOfAirInterfaces(mountName, ltpStructure, uuidUnderTest, requestHeaders, traceIndicatorIncrementer) {
+  let configuredGroupOfAirInterfacesResponse = {};
+  let configuredGroupOfAirInterfaceList = [];
+  try {
+    /****************************************************************************************
+     * extracting list of serving physic ltps for air-interface uuidUnderTest
+     ****************************************************************************************/
+    let firstLayerToReachEthernetContainer = [PURE_ETHERNET_STRUCTURE.MODULE + PURE_ETHERNET_STRUCTURE.LAYER_PROTOCOL_NAME, HYBRID_MW_STRUCTURE.MODULE + HYBRID_MW_STRUCTURE.LAYER_PROTOCOL_NAME];
+    let secondLayerToReachEthernetContainer = [ETHERNET_CONTAINER.MODULE + ETHERNET_CONTAINER.LAYER_PROTOCOL_NAME];
+    let interfaceListToReachEthernetContainer = [firstLayerToReachEthernetContainer, secondLayerToReachEthernetContainer];
+    let airInterfaceLtpUnderTest = await LtpStructureUtility.getLtpForUuidFromLtpStructure(uuidUnderTest, ltpStructure);
+    let clientContainerLtp = await LtpStructureUtility.getHierarchicalClientLtpForInterfaceListFromLtpStructure(airInterfaceLtpUnderTest, interfaceListToReachEthernetContainer, ltpStructure);
+    if (Object.keys(clientContainerLtp).length > 0) {
+      let servingPhysicLtpList = await getServingPhysicLtpList(clientContainerLtp, ltpStructure);
+      if (servingPhysicLtpList != undefined || servingPhysicLtpList.length != 0) {
+        for (let i = 0; i < servingPhysicLtpList.length; i++) {
+          let configuredResource = {};
+          let servingPhysicLtp = servingPhysicLtpList[i];
+          let layerProtocolName = servingPhysicLtp[onfAttributes.LOGICAL_TERMINATION_POINT.LAYER_PROTOCOL][0][onfAttributes.LAYER_PROTOCOL.LAYER_PROTOCOL_NAME];
+          let airInterfaceLayerProtocolName = AIR_INTERFACE.MODULE + AIR_INTERFACE.LAYER_PROTOCOL_NAME;
+          let wireInterfaceLayerProtocolName = WIRE_INTERFACE.MODULE + WIRE_INTERFACE.LAYER_PROTOCOL_NAME;
+          /****************************************************************************************
+          * extracting link-id if radio-link-bonding
+          ****************************************************************************************/
+          if (layerProtocolName == airInterfaceLayerProtocolName) {
+            let linkIdResponse = await getAirInterfaceConfigurationLinkId(mountName, servingPhysicLtp, requestHeaders, traceIndicatorIncrementer);
```

Comment:
traceIndicatorIncrementer is not post incremented. Please have a check is it fine.